libs/protocol-ui/src/components/molecules/ControlledDatePicker/ControlledDatePickerButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {
+  Button,
+  ButtonProps,
+  forwardRef,
+  ComponentWithAs,
+} from '@chakra-ui/react';
+import { HiOutlineCalendar } from 'react-icons/hi';
+
+export type ControlledDatePickerButtonProps = ButtonProps & {
+  value?: React.ReactNode;
+  onClick?: () => void;
+  ref?: HTMLInputElement;
+  buttonDisabled?: boolean;
```

Comment:
Isn't disabled a prop in ButtonProps?

Reply:
Yeah -- I had an issue with a conflict when passing it in, but will revisit that and try again since this is a bit strange

Reply:
Also we can remove this completely -- I opted for setting the visibility to hidden instead of disabling the button. I disabled it previously when Custom wasn't selected. Going to remove